docs/fiddles/features/web-bluetooth/main.js

```diff
@@ -14,6 +14,31 @@ function createWindow () {
     } 
   })
 
+  mainWindow.webContents.session.setBluetoothPairingHandler((details, callback) => {
+    const response = {}
+  
+    switch (details.pairingKind) {
+      case 'confirm': {
+        response.confirmed = confirm(`Do you want to connect to device ${details.deviceId}`)
```

Comment:
I don't think `confirm` and `prompt` are defined in the main process. For `confirm`, users could probably use `dialog.showMessageBoxSync` or `dialog.showMessageBox`, but I don't think there's a way to prompt for input with that API, so for `providePin` I think users would need to create a new `BrowserWindow` UI to prompt for that input.

Reply:
ooh good call... I was just trying to make the sample code simple... I'll clean it up.